private-templates-service/README.md

```diff
@@ -1,62 +1,63 @@
+3. Enter in the Azure Active Directory App Registration application (client) id into the '**App_id**' field. 
+4. Enter the id of your App Service Plan into the '**Server_farm_id**' field. 
+5. If 'Yes' is selected for '**Telemetry_opt_in**', we will collect feedback from your instance of ACMS using App Insights. 
+6. Click '**Next**' and '**Deploy**'.
 
-const mongoClient: ClientOptions = {
-  authenticationProvider: new AzureADProvider(),
-  storageProvider: new InMemoryDBProvider(),
-}
-const client: TemplateServiceClient = TemplateServiceClient.init(mongoClient);
-app.use("/template", client.expressMiddleware());
-app.use("/user", client.userExpressMiddleware());
-```
+Using the 'Deploy to Azure' button will fetch an image using the latest published version of [adaptivecards-templating-service](https://www.npmjs.com/package/adaptivecards-templating-service) and [adaptive-templating-service-typescript-node](https://www.npmjs.com/package/adaptive-templating-service-typescript-node). Once the deployment as finished, you will see the admin portal hosted at '**{Sites_adaptivecms_name}**.azurewebsites.net' and be able to hit the endpoints at the same URL. 
 
-**5. In 50 in AzureADProvider.ts, replace `#{CLIENT_ID_TOKEN}#` with your client ID token. Ask a dev for this string! `result = result && "#{CLIENT_ID_TOKEN}#" === decodedToken.payload.aud;`**
+### Creating a new AAD App Registration
 
-**6. Run `npm run dev`. This command concurrently runs the client and server locally. Navigate to `localhost:3000` to see the site.**
+1. Go to the [Azure Portal](portal.azure.com). 
+2. In the searchbar, type 'Azure Active Directory' and select the AAD service. 
+3. Select 'App Registrations' under the 'Manage' header. 
+4. Click 'New registration'.
+5. Fill in the details for a new AAD registration. Under 'Redirect URI', enter the URL at which the admin portal is hosted at from the 'Deploy to Azure' instructions. 
+6. Click 'Register'. 
 
-# Running the frontend and backend of Adaptive Cards CMS Locally
+## Running the latest changes locally with MongoDB
 
-To run this web application locally, you must have the following installed on your system:
+Prerequisites:
 
 -   Git
 -   [Node v12](https://nodejs.org/en/download/)
+-   MongoDB 
+
+1. Clone the repository.
+
+2. Switch to the desired branch. The latest build is on `dev`.
 
-**1. Clone the repository.**
+3. Set the below environment variables to the desired values. 
+
+```
+ACMS_DB_CONNECTION // Mongo database connection string
+ACMS_APP_ID // Azre Active Directory App Registration Application (client) id 
+ACMS_REDIRECT_URI // http://localhost:3000
+```
 
-**2. Switch to the appropriate branch.**
+4. Open `server/app.ts` and insert the following code at line 38. 

```

Comment:
Instead of using a line number, can we add a comment at line 38 in app.ts saying something like `// If running locally, add code specified in ReadMe enabling Access-Control-Allow-Origin here`. 

Step 4. in the readme could then be something like: 
`Open server/app.ts and replace the comment "If running locally, add code specified in ReadMe enabling Access-Control-Allow-Origin here" with the following code`
`